install.sh

```diff
@@ -144,6 +144,18 @@ Install()
     # If update, start Wazuh
     if [ "X${update_only}" = "Xyes" ]; then
         WazuhUpgrade $INSTYPE
+
+        # Update versions previous to Wazuh 4.8
+        if [ "X$INSTYPE" = "Xserver" ]; then
+            local OSSEC_CONFIGURATION_FILE="$USER_DIR/etc/ossec.conf"
+            local VULN_TEMPLATE="./etc/templates/config/generic/wodle-vulnerability-detection.manager.template"
+            local INDEXER_TEMPLATE="./etc/templates/config/generic/wodle-indexer.manager.template"
+
+            # Load vulnerability detector replacer commands
+            . ./src/init/replace_vulnerability_detector.sh
+            updateVulnerabilityDetector $OSSEC_CONFIGURATION_FILE $VULN_TEMPLATE $INDEXER_TEMPLATE
```

Comment:
Nitpick (naming): I think we should align the names and make them more expressive, like
```suggestion
            local OSSEC_CONF_PATH="$USER_DIR/etc/ossec.conf"
            local VULN_TEMPLATE_PATH="./etc/templates/config/generic/wodle-vulnerability-detection.manager.template"
            local INDEXER_TEMPLATE_PATH="./etc/templates/config/generic/wodle-indexer.manager.template"

            # Load vulnerability detector replacer commands
            . ./src/init/replace_vulnerability_detector.sh
            updateVulnerabilityDetector $OSSEC_CONF_PATH $VULN_TEMPLATE_PATH $INDEXER_TEMPLATE_PATH
```

Reply:
done